### PR TITLE
Accessibility changes to announce Sync complete event

### PR DESCRIPTION
**Pull request by pxwxnvermx**
## Product Description
Added aria-live region to make notification announcements. This can be further used for other notification across the platform which need to be announced for people who use screen-readers to navigate HQ. 

https://user-images.githubusercontent.com/31195143/214783650-6a0b345b-74ff-4126-9d77-ebe35e59f66f.mp4

## Technical Summary
[Ticket Link](https://dimagi-dev.atlassian.net/browse/USH-1934)

### Safety story
Tested locally with Voiceover screen-reader

### QA Plan
https://dimagi-dev.atlassian.net/browse/QA-4754

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Comment by Robert-Costello** [comment]
This is great! The changes look good to me, but I would suggest sending this one through QA.

**Review by stephherbers — APPROVED** [review]
Fantastic improvement, but also note Robert's comment

File: corehq/apps/cloudcare/static/cloudcare/js/utils.js [hunk 1]
```diff
@@ -136,15 +136,21 @@ hqDefine('cloudcare/js/utils', [
         }
     };
 
+    var updateScreenReaderNotification = function (notificationText) {
+        $('#sr-notification-region').text("");
```
Comment (stephherbers) [inline corehq/apps/cloudcare/static/cloudcare/js/utils.js:140]: What is the purpose of this?

Comment (pxwxnvermx) [reply corehq/apps/cloudcare/static/cloudcare/js/utils.js:140]: this resets the live region with blank text before updating with the notification text, when the notification text was same it was not getting read by screen-reader on multiple clicks of "Sync" button, this is a fix to that problem.

**Review by Robert-Costello — APPROVED** [review]

**Pull request merged**